src/modules/LoggedIn/Project/ProjectList.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react'
+import { Card, Row, Col } from 'antd';
+import 'antd/dist/antd.css';
+import { EllipsisOutlined } from '@ant-design/icons';
+const { Meta } = Card;
 
 const ProjectList = () => {
-  return <div>Project List</div>
+  const [card1, setCard1] = React.useState([
```

Comment:
nama state nya jangan card1 ya, diganti projects aja. Biar lebih jelas dia isinya apa
```suggestion
  const [projects, setProjects] = React.useState([
```